unity-renderer/Assets/DCLPlugins/ECS7/Systems/RaycastSystem/ECSRaycastSystem.cs

```diff
@@ -205,11 +208,11 @@ private Ray CreateRay(IParcelScene scene, IDCLEntity entity, PBRaycast model)
             return ray;
         }
 
-        private DCL.ECSComponents.RaycastHit CreateSDKRaycastHit(IParcelScene scene, PBRaycast model, RaycastHit unityRaycastHit, KeyValuePair<IDCLEntity, uint>? hitEntity, Vector3 globalOrigin)
+        private RaycastHit CreateSDKRaycastHit(IParcelScene scene, PBRaycast model, UnityEngine.RaycastHit unityRaycastHit, KeyValuePair<IDCLEntity, uint>? hitEntity, Vector3 globalOrigin)
         {
             if (hitEntity == null) return null;
 
-            DCL.ECSComponents.RaycastHit hit = new DCL.ECSComponents.RaycastHit();
+            RaycastHit hit = new RaycastHit();
```

Comment:
+1

Reply:
I've added a `TODO` there to revisit it later, since it's hard to track the lifespan of that instance... It could leave until the scene request messages or it could be dropped for a new message.
So it will take some time, since it's not that much critical, I will prefer to leave for another PR while I focus on fixing allocation on components deserialization which are a much worse than this one :S